docs/source/installation/login.md

```diff
@@ -1,40 +1,48 @@
 # Login
 
 [Keycloak](https://www.keycloak.org/) is the name of the open-source user management software that's automatically deployed within QHub. It's used to store the database of all
-users in your QHub instance, and can provide connectivity to other services such as GitHub/Auth0 single-sign-on.
+users in your QHub instance, and can provide connectivity to other services such as GitHub/Auth0 single sign-on.
 
-If you ran `qhub init` to create your `qhub-config.yaml` configuration file as detailed in the [Usage guide](usage.md), you will have been provided with a
-`random password for Keycloak root user`. **The password will also be visible in the `qhub-config.yaml` file under the `security.keycloak.initial_root_password` field**.
-
-The `root` Keycloak user is only able to login and manage the Keycloak identity management section of QHub. It's not a user of the wider QHub data science platform.
+As with many user admin tools, in Keycloak you start with an initial login (`root`) that has the ability to administer and create new users. The `root` user is a Keycloak-specific user. It can only be used to login and manage the Keycloak identity management section of QHub. It's not a user of the wider QHub data science platform.
 
 ## Change Keycloak root password
 
+`root`'s password is generated by the `qhub init` command. If you ran this command while following the steps under [Usage](usage.md), you should have seen something like the following in the output:
+
+```
+Securely generated default random password=<alphanumeric-string> for Keycloak root user
+```
+
+The init command also saves the root user password to your `qhub-config.yaml` configuration file under the following path:
+
+`security.keycloak.initial_root_password`
+
 After the initial deployment, it is **highly** recommended that you change the Keycloak `root` user password as soon as you can.
 
+> NOTE: Once you change the root password you will not be able to [add users from the command line](login.md#add-user-from-the-command-line)
```

Comment:
Why change the root password? Is it because the yaml config file gets checked in to version control?